app/workers/update_taxon_worker.rb

```diff
@@ -2,13 +2,16 @@ class UpdateTaxonWorker
   include Sidekiq::Worker
 
   def perform(content_id, attributes)
-    taxon = Taxonomy::BuildTaxon.call(content_id: content_id)
-    taxon.assign_attributes(attributes)
+    previous_taxon = Taxonomy::BuildTaxon.call(content_id: content_id)
+    updated_taxon = previous_taxon.clone
+    updated_taxon.assign_attributes(attributes)
 
-    payload = Taxonomy::BuildTaxonPayload.call(taxon: taxon)
+    Taxonomy::SaveTaxonVersion.call(updated_taxon, 'Bulk update', previous_taxon: previous_taxon)
```

Comment:
It might be good to make the "Bulk update" parameter, a optional argument to this worker, so that it can be set to a more specific message where the worker is used. Just a suggestion, no need to implement this straight away.